### PR TITLE
Add fellows reflections blog

### PR DESCRIPTION
**Pull request by pherterich**
Add image

### CODE REVIEW

**Comment by pherterich** [comment]
Pinging @NPDebs to see if I've broken anything and to hand to Berenice to merge when ready.

File: _posts/2023-08-29-reflections-of-a-resident-fellow.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Reflections of a resident fellow
+authors:
+- pherterich
+image: /images/DSC_9454.jpg
+photos:
+  name: Photo by Software Sustainability Institute
```
Comment (NPDebs) [inline _posts/2023-08-29-reflections-of-a-resident-fellow.md:8]: Patri, I hope it's okay...I added the photo credit beside the image.

File: _posts/2023-08-29-reflections-of-a-resident-fellow.md [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
 ---
 
 In October 2022, I was one of the lucky people invited to explore a Resident Fellowship with OLS. In the announcement blog, I claimed that I wanted to achieve the following with my Fellowship:
 
 *“Patricia will work on a new module on open source software and growing the OLS fellows community. She also plans to explore consultancy services and is always up for well-facilitated and interesting virtual events.”*
 
-So 10 months later, how did it go? What have I achieved and learned? This blog summarises my Resident Fellow journey.
+So 10 months later, how did it go? What have I achieved and learned? This blog summarises my **Resident Fellow journey**.
```
Comment (NPDebs) [inline _posts/2023-08-29-reflections-of-a-resident-fellow.md:15]: This was made bold to help readers conceptualise the post easier.

**Review by NPDebs — APPROVED** [review]
Good news. You haven't broken anything! 😄

The posts looks good to be merged. 🎉 Brilliant choice of image, by the way. 🤍

Awaiting approval from @bebatut. 🙌🏼

**Comment by pherterich** [comment]
Thanks for improving things - I knew internal links can be made smarter but that's when I usually start breaking stuff 😆 - @bebatut happy for this to get merged whenever you're okay with it.

**Review by yochannah — APPROVED** [review]

**Pull request merged**